packages/backend/src/document/document.dto.ts

```
@@ -10,7 +10,7 @@ import {
 } from 'class-validator';
 import { VALID_FILE_ID_PATTERN } from '../file/file.constants';
 
-const DOCUMENT_TYPES = ['sheet', 'doc', 'slides', 'pdf', 'note', 'image'] as const;
+const DOCUMENT_TYPES = ['sheet', 'doc', 'slides', 'pdf', 'note', 'image', 'board'] as const;
```

ESLint

[error] Replace `'sheet',·'doc',·'slides',·'pdf',·'note',·'image',·'board'` with `⏎··'sheet',⏎··'doc',⏎··'slides',⏎··'pdf',⏎··'note',⏎··'image',⏎··'board',⏎`

(prettier/prettier)